api/openapi.yaml

```diff
@@ -26,6 +26,11 @@ paths:
           schema:
             type: integer
             default: 0
+        - in: query
+          name: query
+          description: "query to search tests"
```

Comment:
Minor: maybe it's a good idea to specify in which attributes the search is based on. It would be great to have the search using the `name`, `description` and `url` of the test.

Reply:
this is good idea!

Reply:
I've added description, but URL is very trigger specific, so I think for this basic implementation it should be enough